include/lsst/afw/math/BoundedField.h

```diff
@@ -85,6 +88,20 @@ class BoundedField : public table::io::PersistableFacade<BoundedField>, public t
     ) const;
 
     /**
+     * @brief      Compute the integral of this function over its bounding-box.
+     *
+     * @return     The value of the integral.
+     */
```

Comment:
I don't think we normally do all these extra spaces between the Doxygen tag and the text they correspond to.  I also don't think the `@brief` tags are actually necessary; Doxygen will automatically treat the first line as the brief summary.